### PR TITLE
Cover zeek-client updates in NEWS for Zeek 5.2 [skip ci]

### PR DESCRIPTION
**Pull request by ckreibich**
This is the `NEWS` equivalent of zeek/zeek-docs#173. It'll need cherry-picking into `release/5.2`.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -292,6 +292,14 @@ Changed Functionality
   updated to match the current standardized set of languages. Unknown layout
   values now attempt to fallback to a "parent" layout if one is available.
 
+- In the cluster management framework, the controller now supports Broker's
+  WebSocket data transport for communication with clients. It listens on TCP
+  port 2149 for this purpose. zeek-client now likewise uses the WebSocket
+  transport, removing its runtime dependency on the Broker library and enabling
+  standalone installation. The client still bundles with Zeek by default but is
+  now also available on PyPI and installable via `pip install zeek-client`.
```
Comment (awelzel) [inline NEWS:300]: May need double backticks?

**Review by awelzel — APPROVED** [review]

File: NEWS (hunk 1 above)
Comment (ckreibich) [reply NEWS:300]: Indeed! Sigh.

**Comment by ckreibich** [comment]
I was surprised that CI ran for this, given the `[skip ci] tag on the commit. I checked the [Cirrus docs](https://cirrus-ci.org/guide/writing-tasks/#conditional-task-execution) and they say

> If you open a PR, PR title will be checked for [skip ci] or [ci skip] instead of the last commit message on the PR branch.

I'm going to try that here.

**Comment by ckreibich** [comment]
Worked as advertised.

**Pull request merged**